tasks/README.md

```diff
@@ -6,4 +6,9 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task449_app_review_classification` | Classification of app_reviews review | Classification
```

Comment:
Update the summary field with more details like
One Suggestion 
Given a ....... the task is to ...........